lib/tasks/create_application_status_graph.rake

```diff
@@ -0,0 +1,5 @@
+task :create_application_status_graph => :environment do
+  application = CandidateApplication.create!
+  AASMDiagram::Diagram.new(application.aasm, 'tmp/application.png')
+  application.delete
+end
```

Comment:
Maybe we do not need to create and delete the application (`.new` should be enough):
```
  application = CandidateApplication.new
  AASMDiagram::Diagram.new(application.aasm, 'tmp/application.png')
```

Reply:
yup - fixed